src/library-authoring/library-detail/specs/LibraryAuthoringPage.spec.jsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { screen } from '@testing-library/react';
+import { waitFor } from '@testing-library/dom';
+import { cleanUp, ctxRender, expectAction } from '../../common/specs/helpers';
+import {
+  blockStateFactory,
+  libraryFactory,
+} from '../../common/specs/factories';
+import LibraryAuthoringPageContainer from '../LibraryAuthoringPage';
+import { STORE_NAMES } from '../../common/data';
+
```

Comment:
Could you add a test for the `Add {blocktype}` button?

Reply:
This test has been written, but it's in the complex authoring PR, where I added a bunch more helpers to allow more comprehensive and useful introspection during the tests.